uuid.js

```diff
@@ -11,6 +11,9 @@
   // returns 128-bits of randomness, since that's what's usually required
   var _rng;
 
+  // Allow for MSIE11 msCrypto
+  var _crypto = _global.crypto || _global.msCrypto;
```

Comment:
The MDN page you referenced recommends the order I proposed, as does this MS page:
https://msdn.microsoft.com/en-us/library/ie/dn265046%28v=vs.85%29.aspx

    var crypto = window.crypto || window.msCrypto;